packages/vue/scripts/postinstall.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+
+const isDependency = __dirname.includes("node_modules");
+
+function runPostInstall() {
+  createPlaygroundVue();
+}
+
+function createPlaygroundVue() {
+  const playgroundPath = path.resolve(__dirname, "..", "src/Playground.vue");
+  if (!fs.existsSync(playgroundPath) && !isDependency) {
+    fs.writeFileSync(playgroundPath, getPlaygroundContent());
+  }
+}
+
+function getPlaygroundContent() {
+  return `<template>
+  <div id="playground"></div>
+</template>
+
+<script>
+// Use this component to play with other components
```

Comment:
@EduardDopler taking advantage of this pr, wdyt about importing some component by default on `Playground.vue` (as example), maybe `SfHeading` with a _Hello Sfui_? :)

Reply:
Funny, had the same idea before :D I just wanted to keep this PR as small as possible, it being a bugfix not a feature.
But yeah, I agree!